app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: "Cafe") -> None:
```

Comment:
The function `go_to_cafe` should return a string in all cases. Currently, if no exceptions are raised and `masks_to_buy` is 0, the function implicitly returns `None`. Consider adding a return statement to handle this case and ensure a string is returned as required by the task description.